libs/stache/schematics/ng-add/ng-add.schematic.ts

```diff
@@ -31,16 +28,8 @@ function installEssentialSkyUxPackages(skyuxVersion: string): Rule {
 
 export default function ngAdd(): Rule {
   return async (_tree, context) => {
-    // Get the preferred version of SKY UX found in the "peerDependencies" section of
-    // @blackbaud/skyux-lib-stache package.json.
-    const packageJson = fs.readJsonSync(
-      path.resolve(__dirname, '../../package.json'),
-    );
-
-    const skyuxVersion = packageJson.peerDependencies['@skyux/core'];
-
     context.addTask(new NodePackageInstallTask());
 
-    return chain([installEssentialSkyUxPackages(skyuxVersion)]);
+    return chain([installEssentialSkyUxPackages('^11.0.0-alpha.6')]);
```

Comment:
Could we drop `fs-extra` without having to bury this hard-coded version?

Reply:
This value is tested against the value of `VERSION.full` in the unit test, so the test should fail whenever we update the dependency without also updating the version specified, here. I thought it was an okay compromise to having `fs-extra` as a peer dependency.